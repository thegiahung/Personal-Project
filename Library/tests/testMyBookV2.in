Add collection mybooks.csv
number copies
add member Python
add member Java
add member C
rent 100000 111111
rent 100000 111123
relinquish all 100000
rent 100001 111122
rent 100001 111123
relinquish all 100001
common 100000 100001
rent 100002 111119
rent 100002 111123
relinquish all 100002
common 100000 100001 100002
exit
