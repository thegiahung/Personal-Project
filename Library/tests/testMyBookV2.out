user: 17 books successfully added.

user: Accel World (Reki Kawahara): 1
Ano Hi Mita Hana no Namae o Bokutachi wa Mada Shiranai (Nagai Tatsuyuki): 1
Arifureta: From Commonplace to World's Strongest (Shirakome Ryo): 1
Bladedance of Elementalers (Shimizu Yu): 1
Date A Live (Koshi Tachibana): 1
Gakuen Toshi Asterisk (Miyazaki Yuu): 1
Kaguya-sama (Akasaka Aka): 1
Kanojo Okarashimasu (Miyajima Reiji): 1
Love Live! School Idol Project (Yoshiaki Fujisawa): 1
Overlord (Kugane Maruyama): 1
Rakudai Kishi No Calvary (Misora Riku): 1
Sword Art Online (Reki Kawahara): 1
Takagi (Shin-Ei Animation): 1
The Iruity at Magic High School (Sato Tsutomu): 1
The Misfit of Demon King Academy (Shu): 1
The Rising of the Shield Hero (Yusagi Aneko): 1
Your Lie In April (Fuji TV): 1

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Bladedance of Elementalers (Shimizu Yu)

user: Success.

user: Success.

user: Success.

user: Bladedance of Elementalers (Shimizu Yu)

user: Ending Library process.
